x-pack/test/functional_with_es_ssl/apps/cases/list_view.ts

```diff
@@ -18,104 +19,161 @@ export default ({ getPageObject, getService }: FtrProviderContext) => {
 
   describe('cases list', () => {
     before(async () => {
-      await common.navigateToApp('cases');
-      await cases.api.deleteAllCases();
+      await cases.navigation.navigateToApp();
     });
 
     after(async () => {
       await cases.api.deleteAllCases();
     });
 
-    beforeEach(async () => {
-      await common.navigateToApp('cases');
-    });
-
-    it('displays an empty list with an add button correctly', async () => {
-      await testSubjects.existOrFail('cases-table-add-case');
+    describe('empty state', () => {
+      it('displays an empty list with an add button correctly', async () => {
+        await testSubjects.existOrFail('cases-table-add-case');
+      });
     });
 
-    it('lists cases correctly', async () => {
+    describe('listing', () => {
       const NUMBER_CASES = 2;
-      await cases.api.createNthRandomCases(NUMBER_CASES);
-      await common.navigateToApp('cases');
-      await cases.common.validateCasesTableHasNthRows(NUMBER_CASES);
+
+      before(async () => {
+        await cases.api.createNthRandomCases(NUMBER_CASES);
+        await header.waitUntilLoadingHasFinished();
+        await cases.casesTable.waitForCasesToBeListed();
+      });
+
+      after(async () => {
+        await cases.api.deleteAllCases();
+      });
+
+      it('lists cases correctly', async () => {
+        await cases.casesTable.validateCasesTableHasNthRows(NUMBER_CASES);
+      });
     });
 
-    it('deletes a case correctly from the list', async () => {
-      await cases.api.createNthRandomCases(1);
-      await common.navigateToApp('cases');
-      await testSubjects.click('action-delete');
-      await testSubjects.click('confirmModalConfirmButton');
-      await testSubjects.existOrFail('euiToastHeader');
+    describe('deleting', () => {
+      const NUMBER_CASES = 9;
+
+      before(async () => {
+        await cases.api.createNthRandomCases(NUMBER_CASES);
+        await header.waitUntilLoadingHasFinished();
+        await cases.casesTable.waitForCasesToBeListed();
+      });
+
+      after(async () => {
+        await cases.api.deleteAllCases();
+      });
+
+      it('deletes a case correctly from the list', async () => {
+        await testSubjects.click('action-delete');
+        await testSubjects.click('confirmModalConfirmButton');
+        await testSubjects.existOrFail('euiToastHeader');
+      });
+
+      it('bulk delete cases from the list', async () => {
+        await cases.casesTable.selectAndDeleteAllCases();
+        await cases.casesTable.validateCasesTableHasNthRows(0);
+      });
     });
 
-    it('filters cases from the list with partial match', async () => {
-      await cases.api.deleteAllCases();
-      await cases.api.createNthRandomCases(5);
+    describe('filtering', () => {
```

Comment:
This whole suite of tests as they are, influence each other. First you search with text, then you apply a filter to that same search and then other similar tests build on top of what the previous did which doesn't really test the scenarios we want.